tools/xcodeproj-shims/swiftc-stub.sh

```diff
@@ -7,8 +7,9 @@ if [[ $# -eq 1 && $1 == "-v" ]]; then
     exec swiftc -v
 fi
 
+# Source of json-util: https://github.com/dominictarr/JSON.sh
```

Comment:
an alternative to this is to use python inline:

(something like `python -c` with)
```python
import json

files = set()
for derived_files in json.load(sys.stdin).values(): 
    files.extend(derived_files.values())

for file in files:
    open(file, 'a').close() 
```

or honestly just to rewrite the entire stub to use a language that isn't bash

Reply:
see, for example, https://github.com/lyft/envoy-mobile/pull/615/files#diff-8c6a45719413cc768d2e976b0c540484

Reply:
my first thought is on python then I was like "wait does it then depend on python?" But now that the entire bazel heavily depend on python/starlark. Guess this is after all the viable solution

Reply:
Python is installed by default on macOS, same as bash/sh. Another alternative would be to write a wrapper that gets compiled into a native executable